miio/vacuumcontainers.py

```diff
@@ -167,6 +167,11 @@ def is_on(self) -> bool:
             or self.state_code == 17
         )
 
+    @property
+    def is_water_box_attached(self) -> int:
+        """Return True is water box is installed."""
+        return self.data["water_box_status"] == 1
```

Comment:
This is not always available, so let's avoid crashing just in case someone tries to access it :-)
```suggestion
        return "water_box_status" in self.data and self.data["water_box_status"] == 1
```